recipes/mage-ai/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "mage-ai" %}
+{% set version = "0.4.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 493b2d39663dd2f5a926796fb73b610ef96730a602567888a8c429cd336687c1
+
+build:
+  noarch: python
+  skip_compile_pyc:
+    - "*/templates/*.py"          # These should not (and cannot) be compiled
```

Comment:
Just curious, why?

Reply:
Do you need to ship those? Does it make sense to remove them instead?

Reply:
Yes, they need to be included in the package. They're used as code templates in our app.

Reply:
@ocefpaf Does my reply make sense to you?